webpack.build.js

```diff
@@ -54,10 +56,18 @@ module.exports = {
     new MiniCssExtractPlugin({
       filename: '[name]-[contenthash].css'
     }),
+    new webpack.ProvidePlugin({  // Required for Webpack 5, since it removes Node.js polyfills
+      process: 'process/browser',
+    }),
```

Comment:
Do you need  `process/browser` if `process.env` is defined by the EnvironmentPlugin? I interpreted this as saying that EnvironmentPlugin on its own should be fine to define environment variables for a build:
>Want to use environment variables with `process.env.VARIABLE`? You need to use the DefinePlugin or EnvironmentPlugin to define these variables in the configuration.

https://webpack.js.org/migrate/5/#run-a-single-build-and-follow-advises

Reply:
This is required, yes. We need `new webpack.ProvidePlugin({ process: 'process/browser' })` to solve a runtime issue: _"process is not defined"_ error from `util`, which is used by `counterpart`

Side note though, after some additional testing, it seems that `resolve.fallback.util: require.resolve('util')` might not actually be needed! Going to try a clean reinstall to confirm that.

Reply:
Yes, polyfilling the whole thing is safer since dependencies may be using it, but this can be tested and confirmed by just using the define plugin if you wish.

Reply:
Update: `resolve.fallback.util: require.resolve('util')` (and the corresponding dependency in package.json) is ALSO required. Otherwise, we get a _build_ error with the message: _ERROR in ./node_modules/counterpart/index.js_ , _Module not found: Error: Can't resolve 'util' in ..._

My earlier testing _looked like_ it wasn't needed because I disabled the resolve.fallback, but didn't actually take out the dependency from package.json + do a clean install. I wasn't 100% sure that'd make a difference, but it does. Always test with a clean node_modules directory!

Reply:
`npm ci` is your friend!